packages/slate/test/interfaces/Element/isElement/elementType.tsx

```diff
@@ -0,0 +1,11 @@
+import { Element } from 'slate'
+
+export const input = {
+  type: 'paragraph',
+  children: [{ text: '' }],
+}
+export const test = value => {
+  return Element.isElementType(value, 'heading-large')
+}
+
+export const output = true
```

Comment:
Shouldn't this return `false` since `'heading-large'` is not `'paragraph`'?

Reply:
We should probably add tests for `true` and `false` cases.

Reply:
My bad. Updated and adding some test for the discriminant key usage